package k80;

import oq.p;
import p071kotlin.Metadata;

/* JADX INFO: loaded from: classes6.dex */
@Metadata(d1 = {"\u0000\f\n\u0002\u0018\u0002\n\u0002\u0010\u000e\n\u0002\b\u0003\u001a\u0011\u0010\u0002\u001a\u00020\u0001*\u00020\u0000¢\u0006\u0004\b\u0002\u0010\u0003¨\u0006\u0004"}, d2 = {"Lk80/f;", "", "a", "(Lk80/f;)Ljava/lang/String;", "contract"}, k = 2, mv = {2, 2, 0}, xi = 48)
public final class e {

    @Metadata(k = 3, mv = {2, 2, 0}, xi = 48)
    public static final /* synthetic */ class a {

        /* JADX INFO: renamed from: a, reason: collision with root package name */
        public static final /* synthetic */ int[] f109150a;

        static {
            int[] iArr = new int[f.values().length];
            try {
                iArr[f.STATIC.ordinal()] = 1;
            } catch (NoSuchFieldError unused) {
            }
            try {
                iArr[f.DYNAMIC.ordinal()] = 2;
            } catch (NoSuchFieldError unused2) {
            }
            f109150a = iArr;
        }
    }

    public static final String a(f fVar) {
        int i15 = a.f109150a[fVar.ordinal()];
        if (i15 == 1) {
            return ip.a.f96137b;
        }
        if (i15 == 2) {
            return ip.a.f96138c;
        }
        throw new p();
    }
}
